pygerduty/v2.py

```diff
@@ -71,7 +71,24 @@ def create(self, **kwargs):
         if "requester_id" in kwargs:
             extra_headers["From"] = kwargs.pop("requester_id")
 
-        data[self.sname] = kwargs
+        new_kwargs = {}
+
+        for kwarg_key, kwarg_value in kwargs.iteritems():
+            if kwarg_key.endswith('_id'):
+                new_key = kwarg_key[:-3]
+                new_kwargs[new_key] = {
+                    'id': kwarg_value,
+                    'type': new_key
+                }
+            elif kwarg_key.endswith('_ids'):
```

Comment:
I believe if it ends in `_ids` it will be a list of ids `["ID1", "ID2"]` that needs to be translated to a list of objects 

```
[{"type": "sometype", "id": "ID1"}, ...]
```

This seems to be doing the same thing as `_id`